template.yaml

```diff
@@ -13,15 +13,17 @@ Globals:
         UpdateUserFeatureFlagFunction: feature-flag-backend-UpdateUserFeatureFlagFunction-4uOLMOFjOntY
         GetUserFeatureFlagsFunction: feature-flag-backend-GetUserFeatureFlagsFunction-yqy9D1QsjzOC
         GetUserFeatureFlagFunction: feature-flag-backend-GetUserFeatureFlagFunction-hMf1qBMVQLhi
-        RateLimiterFunction: feature-flag-backend-RateLimiterFunction-4CxLCmGRkXpW
+        RateLimiterFunction: feature-flag-backend-RateLimiterFunction-TUAIVAMf6B2j
 
 Resources:
   HealthCheckFunction:
     Type: 'AWS::Serverless::Function'
+    Metadata:
+      BuildMethod: go1.x
     Properties:
       CodeUri: health-check/
-      Handler: health-check
-      Runtime: go1.x
+      Handler: bootstrap
+      Runtime: provided.al2
```

Comment:
Is is fine to change the handler here?

Reply:
Yes tested the lambdas they were working fine.

Reply:
<img width="813" alt="image" src="https://github.com/Real-Dev-Squad/feature-flag-backend/assets/52795644/6e239094-7909-4580-8a56-56972e7ae149">